packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1032,6 +1032,7 @@ oie.enroll.okta_verify.setup.email.title = Check your email
 oie.enroll.okta_verify.setup.sms.title = Check your text messages
 oie.enroll.okta_verify.setup.skipAuth.subtitle = To set up Okta Verify on additional devices, you can copy an existing Okta Verify account onto a new device.
 oie.enroll.okta_verify.setup.skipAuth.openOv = Open Okta Verify on any of your other Okta Verify devices (<$1>{0}</$1>).
+oie.enroll.okta_verify.setup.skipAuth.openOv.suchAs = Open Okta Verify on any of your other Okta Verify devices (Such as your <$1>{0}</$1>).
```

Comment:
Can you add a comment above this translation to give an example of what the parameter will be? This is to provide context to translators to help them translate the text properly. 

i.e. 
`# Such as your <b>phone</b>`

Reply:
@glenfannin-okta thanks for the review. I already added this in 7.9 branch but since this PR is jut for translation, so i merged to master. I believe it will be there when we backport the branch.